### PR TITLE
Add support for a custom class to modal wrapper

### PR DESCRIPTION
**Pull request by saqimtiaz**
As per #4485 add support for a custom class to modal wrapper, by means of a field in the modal tiddler. The class is added to the modal wrapper in addition to the default class, allowing for custom styling via css of any part of the modal.

### CODE REVIEW

File: core/modules/utils/dom/modal.js [hunk 1]
```diff
@@ -58,6 +58,9 @@ Modal.prototype.display = function(title,options) {
 	this.adjustPageClass();
 	// Add classes
 	$tw.utils.addClass(wrapper,"tc-modal-wrapper");
+	if(tiddler && tiddler.fields && tiddler.fields.class) {
```
Comment (Jermolene) [inline core/modules/utils/dom/modal.js]: There's a test for "tiddler" being falsey on line 40, so you can avoid checking that. I notice there's the same issue on line 107, if you'd like to kindly address that at the same time.

Comment (saqimtiaz) [reply core/modules/utils/dom/modal.js]: Resolved.

**Comment by Jermolene** [comment]
Thanks @saqimtiaz

**Comment by Jermolene** [comment]
Hi @saqimtiaz sorry I just realised that it would be good to have a docs update too! Many thanks

**Pull request merged**